docker/tools/run_services_tests.sh

```diff
@@ -56,6 +56,13 @@ yell check for registration
 # -----------------------------------------------------------------
 # this probably requires additional CCF keys, need to test this
 if [ "$SGX_MODE" == "HW" ]; then
+    if [ ! -f ${XFER}/ccf/keys/memberccf_privk.pem ] ; then
+        die unable to locate CCF policies keys
+    fi
+
+    try cp ${XFER_DIR}/ccf/keys/memberccf_cert.pem ${PDO_LEDGER_KEY_ROOT}/
+    try cp ${XFER_DIR}/ccf/keys/memberccf_privk.pem ${PDO_LEDGER_KEY_ROOT}/
```

Comment:
Yes, these should make `PDO_LEDGER_KEY_ROOT` the authoritative place where to look for the ledger keys -- to be double checked with HW mode tests.

Reply:
umm... it already was.